src/main/webapp/script.js

```diff
@@ -87,13 +86,16 @@ async function getChart() {
     averageContainer.innerHTML = "Average Sentiment Score: " + averageScore;
   });
 }
-
-function getRangeExclusiveEnd(rangeString) {
-  rangeString.trim();
-  return Number(rangeString.substring(rangeString.indexOf(',') + 1, rangeString.length - 1));
-}
  
-function getRangeInclusiveStart(rangeString) {
-  rangeString.trim();
-  return Number(rangeString.substring(1, rangeString.indexOf(',')));
+function toTooltipString(userComments) {
+  return userComments.map(comment => userCommentAsString(comment)).join("<br>"); 
 }
+
+function userCommentAsString(comment) {
+  commentMagnitude = comment.magnitude;  
+  return comment.commentMsg + "<br> Magnitude Score: " + commentMagnitude;
+}
+
+function convertRangeToString(range) {
+  return range.inclusiveStart + " to " + range.exclusiveEnd;  
+}
```

Comment:
I know range has a toString method, can you simply call that?

Reply:
Is it still needed since the range has its own toString()?

Reply:
since it's a js type of the object, i cant call toString() on it. it returns [Object, object]